### DISCUSSION TITLE
Image size placeholders in media snippet?

### DISCUSSION THREAD

**Discussion by jmatthewpryor** [estruyf/vscode-front-matter#267] [Ideas]
Hi

It seems like it would be useful to have access to the media image width and height as snipper placeholders as described [here](https://frontmatter.codes/docs/markdown#insert-your-own-media-snippet)

Is that possible?

Thanks

**Comment by estruyf** [comment]
So apart from the four placeholders, you would like to get `{imageWidth}` and `{imageHeight}`? If so, that enhancement would certainly be possible.

**Reply by estruyf** [reply]
Created an issue for this to track it: https://github.com/estruyf/vscode-front-matter/issues/271

**Reply by jmatthewpryor** [reply]
tested latest beta - works perfectly
